### ISSUE TITLE
TODO: Bitcart roadmap December 2019

### ISSUE THREAD

**Issue by MrNaif2018** [MrNaif2018/bitcart#92]
Last month of 2019!
Bitcart's one year celebration soon!
- [ ] Add multiple coins support and server management settings to admin (MrNaif2018/bitcart-admin#1)
- [ ] Improve docs (MrNaif2018/bitcart-docs#3)
- [ ] Improve checkout design (MrNaif2018/bitcart-frontend#38)
- [ ] Add more examples of usage and new features to SDK(MrNaif2018/bitcart-sdk#3)
- [ ] Tor and using your own node support(MrNaif2018/bitcart-docker#8)
- [ ] Add omni layer support(electrum-omni probably)
Previous month: #84

**Comment by MrNaif2018** [comment]
Happy New Year!
This year was the key, and the first year for BitcartCC.
It all started in May 2019 with my idea of just sharing electrum wrapper I created before, but it grew bigger.
Now it is a solid project, with community, and lots of features+new ones coming.
Thanks to all of you for motivating me doing this!

This month I've improved admin panel, store UI a lot, made it actually ready to be used.
Email notifications, product images, nice control over connected products/images, multicurrency support, product categories, discounts, and many many more!

Special thanks: https://t.me/bitcartcc/1090
